monitoring/global/iam/roles/create_alarms/terragrunt.hcl

```
@@ -0,0 +1,34 @@
+include "root" {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "github.com/terraform-aws-modules/terraform-aws-iam//modules/iam-assumable-role?ref=v5.11.1"
+}
+
+dependency "policy" {
+  config_path = "../../policies/create_alarms"
+}
+
+locals {
+  account_role = {
+    917039226361 : "AWSReservedSSO_AdministratorAccess_d5667b4ba53e9276"
+    165060789769 : "AWSReservedSSO_AdministratorAccess_4387aec2e456a72d"
+    987436120296 : "AWSReservedSSO_AdministratorAccess_d0db2c3624489273"
+    765086940592 : "AWSReservedSSO_AdministratorAccess_c5576577adce5e76"
+    985612346564 : "AWSReservedSSO_AdministratorAccess_25de7fa586ff018d"
```

```suggestion
    987436120296 : "AWSReservedSSO_AdministratorAccess_d0db2c3624489273"
    985612346564 : "AWSReservedSSO_AdministratorAccess_25de7fa586ff018d"
```

should we remove the once that are currently don't set any alarms?

---

lass drin